packages/fast-usdc/README.md

```diff
@@ -1,101 +1,177 @@
-- the proposal builders are in `@agoric/builders` to work with the a3p-integration `build:submissions` script
-- the RunUtils tests are in `@aglocal/boot` to test running them atop a fresh bootstrapped environment
+- a3p integration tests are in the `a3p-integration` top-level package, separate
+    from this workspace
+- the proposal builders are in `@agoric/builders` to work with the
+    a3p-integration `build:submissions` script
+- the RunUtils tests are in `@aglocal/boot` to test running them atop a fresh
+    bootstrapped environment
 
 Over time we can update our tooling to decouple this more from the `packages` directory.
 
-1. Make a3p-integration `build:submissions` script work with arbitrary builder paths, allowing this to be above `@agoric/builders` in the package graph
-2. Export bootstrap testing utilities from `@aglocal/boot`, allowing this to be above `@aglocal/boot` in the package graph
-3. Update CI to support packages that aren't under `packages/`, eg. a top-level `dapps` directory
+1. Make a3p-integration `build:submissions` script work with arbitrary builder
+    paths, allowing this to be above `@agoric/builders` in the package graph
+2. Export bootstrap testing utilities from `@aglocal/boot`, allowing this to be
+    above `@aglocal/boot` in the package graph
+3. Update CI to support packages that aren't under `packages/`, eg. a top-level
+   `dapps` directory
 4. Move this package out of agoric-sdk
 
-# Transaction feed
-
-## Oracles interface
-
-Oracles run off-chain and interact with the contract via an Agoric smart wallet bridge.
-```mermaid
-sequenceDiagram
-    title Becoming an oracle operator
-    participant OW as Operator N<br/>Smart Wallet
-    participant FUC as Fast USDC<br/>Contract Exo
-    participant CE as Core Eval
-
-    CE->>FUC: makeOperatorInvitation()
-    FUC-->>CE: operatorInvitation
-    CE->>+OW: deposit(operatorInvitation)
-
-    Note left of FUC: Off-chain wallet accepts the operator invitation
-
-    OW->>+FUC: offer(operatorInvitation)
-    FUC-->>OW: operator invitationMakers: {SubmitEvidence}
-
-    Note left of FUC: Off-chain watcher detects evidence
-    OW->>+FUC: offer(SubmitEvidence, evidence)
-```
+# Funds flow
 
 ```mermaid
 sequenceDiagram
-    title Receiving evidence
-    participant W as Operator N<br/>Smart Wallet
-    participant A as Operator N<br/>Admin Oexo
-    participant TF as Transaction<br/>Feed
+  actor User as User
+  participant NEA as User Wallet<br>[Browser]
+  participant ETH as CCTP Contract<br>[Ethereum]
+  participant NFA as Noble Forwarding Account<br/>[Noble]
+  participant NC as Noble Chain<br/>[Noble]
+  participant NAR as Noble-Agoric<br/>[Relayer]
+  participant SA as Settlement Account<br/>[Agoric]
+  participant CFA as Contract Fee Account<br/>[Agoric]
+  participant DAR as Destination-Agoric<br/>[Relayer]
+  participant EUD as End User Destination<br/>[Dest Chain]
```

Comment:
Consider including these participants:
- Circle Attestation Service
- CCTP Relayer


Also some nuance I'm not sure is fully captured:
- "Noble Forwarding Account" is also owned by the CCTP Relayer as they actually register it
- CCTP Relayer can be many parties, but only one when `caller` is specified